doc/glossary.rst

```diff
@@ -810,7 +810,17 @@ Class APIs and Estimator Types
         * :term:`predict` if :term:`inductive`
 
     density estimator
-        TODO
+        An :term:`unsupervised` estimation of input density without a labeled response.
+        Most commonly used techniques are `Histograms <https://scikit-learn.org/stable/modules/density.html#density-estimation-histograms>`_,
```

Comment:
We can change the link to be internal, using sphinx links. The target would be for example `.. _density_estimation:` at the top of `doc/modules/density.rst`, and the link syntax is:
```py
:ref:`density_estimation`
or :ref:`a longer link title <density_estimation>`
```